### PR TITLE
Fix clean and clobber targets for fltk and chplvis

### PR DESCRIPTION
**Pull request by tomyhoi**

### CODE REVIEW

File: tools/chplvis/Makefile [hunk 1]
```diff
@@ -56,7 +53,9 @@ OFILES= $(SRCS:.cxx=.o) $(GENSRCS:.cxx=.o)
 all: chplvis
 
 chplvis: $(GENSRCS) $(OFILES) 
-	$(CXX) -o chplvis $(OFILES) $(LDFLAGS) $(LIBS)
+	$$($(FLTK_CONFIG) --cxx)  -o chplvis $(OFILES) \
+	$$($(FLTK_CONFIG) --ldflags) $$($(FLTK_CONFIG) --libs)
```
Comment (gbtitus) [inline tools/chplvis/Makefile]: Think it might be helpful to indent this second line a few spaces, to make it more obvious that it's a continuation of the command starting on the previous line, rather than a new command. (The trailing backslash on the previous line does say this, but someone reading rapidly might not pick that up.)

**Comment by gbtitus** [comment]
Looks good to me. I didn't test it, though.

BTW, you can bring in a participant by mentioning a username, such as @Kyle-B .  My doing so just now will accomplish that. And if you want Kyle (for example) to be the primary reviewer you can set him as the Assignee, over on the right in the web browser view.

File: third-party/fltk/Makefile [hunk 2]
```diff
@@ -32,12 +32,12 @@ $(FLTK_BUILD_DIR):
...
+	-rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
 
 cleanall:
-	cd $(FLTK_BUILD_DIR) && $(MAKE) cleanall
-	rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
+	-cd $(FLTK_BUILD_DIR) && $(MAKE) cleanall
+	-rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
 
 clobber:
 	-rm -rf $(FLTK_BUILD_DIR) $(FLTK_INSTALL_DIR) *~
```
Comment (Kyle-B) [inline third-party/fltk/Makefile]: This isn't related to your changes, but I'm uneasy about the `rm -rf *~`, why is this needed?

**Comment by Kyle-B** [comment]
I'd rather not ignore errors in the Makefile, but this change is fine with me for now.

**Comment by tomyhoi** [comment]
I've modified the Makefile to not generate an error if the directory isn't there using a shell if.   I removed the *~ from the clean rules.   I noticed clean and cleanall were the same and just had cleanall depend on clean.

**Comment by Kyle-B** [comment]
Looks good to me.
